lib/fortify.rb

```diff
@@ -18,15 +17,6 @@ class NotDefinedError < Error; end
   class InvalidUser < Error; end
 
   class << self
-    def policy_scope(klass)
-      klass.instance_eval(&policy(klass).scope_proc) if policy(klass)
-    end
-
-    def policy(record)
-      # TODO: raise a detailed exception when there is no user set
-      Pundit.policy(user, record)
-    end
-
     def enabled!
```

Comment:
I think the `self.method_name` is clearer and would be consistent with your previous refactoring